src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -166,6 +170,11 @@ void ResumptionDataProcessor::on_event(const event_engine::Event& event) {
   const bool successful_resumption =
       status.error_requests.empty() &&
       status.unsuccesfull_vehicle_data_subscriptions_.empty();
+  LOG4CXX_DEBUG(logger_,
+                "error_requests: "
+                    << status.error_requests.size()
+                    << " unsuccesfull_vehicle_data_subscriptions_ "
```

Comment:
@mked-luxoft This log is valid for VI subscriptions. We have buttons, module data subscriptions as well. So your code should take into account another app subscriptions during resumption

Reply:
@mked-luxoft Format your code please